src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0031-View-AcademicTimePeriodDim.sql

```diff
@@ -20,7 +20,7 @@ AS
 			CAST(GradingPeriod.GradingPeriodDescriptorId as VARCHAR),
 			CONVERT(VARCHAR, GradingPeriod.BeginDate, 112)
 		) as AcademicTimePeriodKey,
-		CAST(SchoolYearType.SchoolYear as VARCHAR) as SchoolYearKey,
+		CAST(SchoolYearType.SchoolYear as VARCHAR) as SchoolYear,
```

Comment:
We can't change the existing file. Please create a new file, like `0032-AcademicTimePeriodDim-Change-SchoolYear.sql` or something like that, with the new definition.

If someone has already installed the AMT, then this file will never run and they'll never get the updated code.